.gitignore

```diff
@@ -24,6 +24,32 @@ manifest.mf
 npm-debug.log
 core/src/main/web/app/templates.js
 core/config/hash
-core/src/main/web/vendor/
 core/src/main/web/index.html
 /bin/
+
+core/src/main/web/vendor/bower_components/jquery/test
```

Comment:
the reason why I did not gitignore all of `core/src/main/web/vendor/bower_components/`
is because I want to commit (in source control) the files that we depend on. If we did ignore all of the `bower_components` then to run the app we would always have to `bower i` which would hit the network, thus negating the benefits of vendoring the files.